File: Own Sample Programs/ArmstrongNumber.java
import java.util.*;
public class ArmstrongNumber{

     public static void main(String []args){
         int input = 1634;
         int num = input;
         double sum = 0;
         int count = 0;
         
         while(num!=0){
             num = num / 10;
             count++;
         }
         
         num = input;
         while(num!=0){
             int part = num % 10;
             sum = sum + Math.pow(part,count);
             num = num / 10;
         }
         
         if (sum == input)
           System.out.println("Armstrong Number: "+sum);
         else
           System.out.println("Not Armstrong Number: "+sum);
     }
}

File: Own Sample Programs/ArmstrongNumber.py
inp = 1634
num = inp
dup = inp
sum = 0
no_of_digits = 0

while dup > 0:
  dup = int(dup / 10)
  no_of_digits = no_of_digits + 1

while num > 0:
  part = num % 10
  print(part)
  val = (part**no_of_digits)
  sum = sum + val
  print('SUM = {} and VAL = {}\n'.format(sum,val))
  num = int(num / 10)

if sum == inp:
  print("Armstrong Number:",sum)
else:
  print("Not Armstrong Number:",sum)